manticore/native/cpu/abstractcpu.py

```diff
@@ -294,6 +294,15 @@ def get_arguments(self):
         """
         raise NotImplementedError
 
+    def get_result_reg(self):
+        """
+        Extract the location a return value will be written to. Produces
+        a string describing a register where the return value is written to.
+        :return: return register name
+        :rtype: string
+        """
+        raise NotImplementedError
```

Comment:
Does mypy give an error if you put the return type like this?
```suggestion
    def get_result_reg(self) -> str:
        """
        Extract the location a return value will be written to. Produces
        a string describing a register where the return value is written to.
        :return: return register name
        :rtype: string
        """
        raise NotImplementedError
```
Could you also add explicit return types to the functions that override this? I'm not sure how smart mypy is or if it even checks untyped definitions.

Looks like there's an options, but I think we will have lots of errors if we turn this on https://mypy.readthedocs.io/en/stable/command_line.html#untyped-definitions-and-calls

Reply:
I didn't get an error :)